adafruit_ccs811.py

```diff
@@ -44,13 +44,16 @@
 __version__ = "0.0.0-auto.0"
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_CCS811.git"
 
+_BASELINE_PACKING_FORMAT = "<H"
```

Comment:
you can remove this line - it uses more memory than just having <H in both spots

Reply:
removed